eZ/Publish/Core/Persistence/Legacy/Content/Location/Gateway/DoctrineDatabase.php

```diff
@@ -355,6 +373,19 @@ public function moveSubtreeNodes(array $sourceNodeData, array $destinationNodeDa
         }
     }
 
+    private function isHiddenByParent($pathString, array $rows)
+    {
+        $parentNodeIds = explode('/', trim($pathString, '/'));
+        array_pop($parentNodeIds);// remove self
+        foreach ($rows as $row) {
+            if ($row['is_hidden'] &&  in_array($row['node_id'], $parentNodeIds)) {
```

Comment:
This check fails because `is_hidden` is not in the select list here: https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/Persistence/Legacy/Content/Location/Gateway/DoctrineDatabase.php#L289

Reply:
@pspanja You'll report an issue, right? cc @andrerom

Reply:
Here it is: https://jira.ez.no/browse/EZP-30147